microsoft-edge/webview2/release-notes/about.md

```diff
@@ -25,14 +25,26 @@ See also:
 <!-- ====================================================================== -->
 ## Phases of adding APIs
 
-New APIs are added in phases, as follows:
-1. APIs are initially introduced as Experimental APIs in a Prerelease SDK package.
-1. Then they become Stable APIs in a Prerelease SDK package.
-1. Soon after, they become Stable APIs in a Release SDK package.  Sometimes, APIs are simultaneously promoted to Stable in a Prerelease SDK and Stable in a Release SDK.
+New APIs are added in three phases, as follows:
+1. APIs are initially introduced as Experimental APIs in a Prerelease SDK package.  In Win32, the type name includes the word "Experimental".
+1. Then the APIs become Stable APIs in a Prerelease SDK package.  In Win32, the type name omits the word "Experimental".
+1. Soon after, the APIs become Stable APIs in a Release SDK package.  Sometimes, APIs are simultaneously promoted to Stable in a Prerelease SDK and Stable in a Release SDK.
 
 ![Diagram of phases of introducing new APIs](./about-images/phases-of-adding-apis.png)
 <!-- .png is used by webview2/release-notes/about.md and webview2/concepts/versioning.md -->
 
+Shorthand names of the phases:
+
+1. Experimental in Prerelease
+1. Stable in Prerelease
+1. Stable in Release
+
+Release Notes sections use this wording for the phases:
+
+1. The following Experimental APIs have been added in this Prerelease SDK.
+1. The following APIs have been promoted from Experimental to Stable in this Prerelease SDK.
+1. The following APIs have been promoted to Stable and are now included in this Release SDK.
```

Comment:
mis-worded.  the APIs were already Stable.  They changed from Stable in Prerelease to Stable in Release.

Reply:
clarified wording in files